fabfile.py

```diff
@@ -127,6 +129,43 @@ def listdir(path):
 
 
 @task
+def create_local_ini():
+    """Replace the local.ini file with one composed from the current .rc file"""
+    random_ini_path = os.path.join(env.projectpath, 'random.ini')
+    local_ini_path = os.path.join(env.projectpath, 'local.ini')
+    if exists(local_ini_path):
+        run('cp %s %s.%d' % (
+            local_ini_path, local_ini_path, int(time())))
+
+    if env.host_string == 'localhost':
+        # The easy case
+        venv("python assembl/scripts/ini_files.py compose -o local.ini " + env.rcfile)
+    else:
```

Comment:
I don't understand the difference of treatments between localhost case and remote server case